src/test/resources/application.conf

```diff
@@ -39,3 +39,8 @@ mysql {
     password = "rawls-test"
   }
 }
+
+liquibase {
+  changelog = "org/broadinstitute/dsde/rawls/liquibase/changelog.xml"
+  useForTests = true
```

Comment:
Flip this switch if you can't liquibase for some reason.  You'll likely have to recreate your MySQL Docker container.